### PR TITLE
goldeneye.py: add page

### PR DESCRIPTION
**Pull request by ni-ui**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
 1.2.0

### CODE REVIEW

File: pages/linux/goldeneye.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# goldeneye
+
+> A http DoS test tool.
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:3]:
```suggestion
> A HTTP DoS test tool.
```
 We can use this in order to better align with the main page.

File: pages/linux/goldeneye.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{website_url}}"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:8]:
```suggestion
`./goldeneye.py {{url}}"`
```

File: pages/linux/goldeneye.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
+
+`./goldeneye.py {{website_url}} -u {{100}} -s {{200}}"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:12]:
```suggestion
`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
```

File: pages/linux/goldeneye.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a specific website without verifying ssl certficate:
+
+`./goldeneye.py {{website_url}} -n"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:16]:
```suggestion
`./goldeneye.py {{url}} -n"`
```

File: pages/linux/goldeneye.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a specific website in debug mode:
+
+`./goldeneye.py {{website_url}} -d"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:20]:
```suggestion
`./goldeneye.py {url}} -d"`
```

File: pages/linux/goldeneye.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display help:
+
+`./goldeneye.py -h"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:24]:
```suggestion
`./goldeneye.py -h`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/goldeneye.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
+
+`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:12]:
```suggestion
`./goldeneye.py {{url}} --useragents {{100}} --sockets {{200}}"`
```
We prefer longer arguments in tldr, whenever possible we add mnemonics in command descriptions to highlight the short arguments.

File: pages/linux/goldeneye.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# goldeneye
+
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:10]:
```suggestion
- Test a specific website with 100 [u]ser agents and 200 concurrent [s]ockets:
```

File: pages/linux/goldeneye.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
+
+- Test a specific website without verifying ssl certficate:
+
+`./goldeneye.py {{url}} -n"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:16]:
```suggestion
`./goldeneye.py {{url}} --nosslcheck"`
```

File: pages/linux/goldeneye.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}"`
+
+- Test a specific website with 100 user agents and 200 concurrent sockets:
+
+`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
+
+- Test a specific website without verifying ssl certficate:
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:14]:
```suggestion
- Test a specific website without [n] verifying the SSL certificate:
```

File: pages/linux/goldeneye.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+`./goldeneye.py {{url}} -n"`
+
+- Test a specific website in debug mode:
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:18]:
```suggestion
- Test a specific website in [d]ebug mode:
```

File: pages/linux/goldeneye.md [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+`./goldeneye.py {url}} -d"`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:20]:
```suggestion
`./goldeneye.py {url}} --debug"`
```

File: pages/linux/goldeneye.md [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+
+- Display help:
+
+`./goldeneye.py -h`
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:24]:
```suggestion
`./goldeneye.py --help`
```

File: pages/linux/goldeneye.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+- Test a specific website without verifying ssl certficate:
+
+`./goldeneye.py {{url}} -n"`
+
+- Test a specific website in debug mode:
+
+`./goldeneye.py {url}} -d"`
+
+- Display help:
```
Comment (kbdharun) [inline pages/linux/goldeneye.md:22]:
```suggestion
- Display [h]elp:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/goldeneye.md (hunk 7 above)
Comment (EmilySeville7cfg) [inline pages/linux/goldeneye.md:12]: The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --useragents 100 --sockets 200`
```

- Unbalanced trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Redundant curly braces for integer literals 100 and 200.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).
- This fix is optional: no style guide recommendation exists for this case.

File: pages/linux/goldeneye.md (hunk 8 above)
Comment (EmilySeville7cfg) [reply pages/linux/goldeneye.md:10]: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

File: pages/linux/goldeneye.md (hunk 10 above)
Comment (EmilySeville7cfg) [reply pages/linux/goldeneye.md:14]: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

File: pages/linux/goldeneye.md (hunk 9 above)
Comment (EmilySeville7cfg) [inline pages/linux/goldeneye.md:16]: The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --nosslcheck`
```

- Unbalanced trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).

File: pages/linux/goldeneye.md (hunk 11 above)
Comment (EmilySeville7cfg) [reply pages/linux/goldeneye.md:18]: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

File: pages/linux/goldeneye.md (hunk 12 above)
Comment (EmilySeville7cfg) [inline pages/linux/goldeneye.md:20]: The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --debug`
```

- Unbalanced closing `}` and trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).

File: pages/linux/goldeneye.md (hunk 14 above)
Comment (EmilySeville7cfg) [reply pages/linux/goldeneye.md:22]: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes were requested because several issues were detected.
- Some review comments address style guide problems.
- This PR should be reviewed by `goldeneye` users.
- Thanks for a contribution. 😸

**Comment by ni-ui** [comment]
Have committed the requested changes @EmilySeville7cfg

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/linux/goldeneye.md (hunk 10 above)
Comment (navarroaxel) [reply pages/linux/goldeneye.md:14]:
```suggestion
- Test a specific website without verifying SSL certificate:
```

**Comment by EmilySeville7cfg** [comment]
@ni-ui, please fix quoting in the firsy example.

File: pages/linux/goldeneye.md [hunk 15]
```diff
@@ -0,0 +1,24 @@
+# goldeneye
```
Comment (navarroaxel) [inline pages/linux/goldeneye.md:1]: Page name should match the command name, please rename the file to `goldeneye.py.md` too
```suggestion
# goldeneye.py
```

Comment (navarroaxel) [inline pages/linux/goldeneye.md:1]: Page name should match the command name, please rename the file to `goldeneye.py.md` too
```suggestion
# goldeneye.py
```

**Comment by ni-ui** [comment]
> @ni-ui, please fix quoting in the first example.

Fixed.

File: pages/linux/goldeneye.md [hunk 16]
```diff
@@ -0,0 +1,24 @@
+# goldeneye.py
```
[the remark this answers is not in the document]
Comment (ni-ui) [reply pages/linux/goldeneye.md]: Please check now

**Review by kbdharun — APPROVED** [review]

File: pages/linux/goldeneye.py.md [hunk 17]
```diff
@@ -0,0 +1,24 @@
+# goldeneye.py
+
+> A HTTP DoS test tool.
+> More information: <https://github.com/jseidl/GoldenEye>.
+
+- Test a specific website:
+
+`./goldeneye.py {{url}}`
```
Comment (navarroaxel) [inline pages/linux/goldeneye.py.md:8]: Is the protocol needed or optional? Perhaps I recommend a full domain here
```suggestion
`./goldeneye.py {{https://example.com}}`
```

**Comment by kbdharun** [comment]
With the flags updated and SSL capitalized this PR would be GTG

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**